clipper_admin/clipper_admin/docker/logging/fluentd.py

```diff
@@ -1,41 +1,45 @@
 import random
 import tempfile
 import os
+import logging
 
 from clipper_admin.container_manager import (
     CLIPPER_INTERNAL_FLUENTD_PORT, CLIPPER_FLUENTD_CONFIG_LABEL,
     CLIPPER_DOCKER_PORT_LABELS
 )
 
 
-FLUENTD_VERSION = 'v1.3-debian-1'  # TODO needs to be update to receive env variable like prometheus
+FLUENTD_REGISTRY            = 'clipper/fluentd'
```

Comment:
Please follow the `__registry__` convention as in https://github.com/ucbrise/clipper/blob/df96eb4924809fedc5a70ef1bc080129c2bf962c/clipper_admin/clipper_admin/deployers/python.py#L199

Reply:
Fixed!